engine/src/main/scala/cromwell/engine/io/IoActor.scala

```diff
@@ -191,26 +201,27 @@ object IoActor {
     // For now explicitly lists 503 as a retryable code here to work around that.
     503
   )
-  
+
   // Error messages not included in the list of built-in GCS retryable errors (com.google.cloud.storage.StorageException) but that we still want to retry
   val AdditionalRetryableErrorMessages = List(
     "Connection closed prematurely"
   ).map(_.toLowerCase)
-  
+
   /**
     * Failures that are considered retryable.
     * Retrying them should increase the "retry counter"
     */
   def isRetryable(failure: Throwable): Boolean = failure match {
-    case gcs: StorageException => gcs.isRetryable || 
+    case gcs: StorageException => gcs.isRetryable ||
       isRetryable(gcs.getCause) ||
-      AdditionalRetryableHttpCodes.contains(gcs.getCode) || 
+      AdditionalRetryableHttpCodes.contains(gcs.getCode) ||
       AdditionalRetryableErrorMessages.exists(gcs.getMessage.toLowerCase.contains)
     case _: SSLException => true
     case _: BatchFailedException => true
     case _: SocketException => true
     case _: SocketTimeoutException => true
     case ioE: IOException if Option(ioE.getMessage).exists(_.contains("Error getting access token for service account")) => true
+    case ioE: IOException => isGCS500(ioE) || isGCS503(ioE)
```

Comment:
These two feel like they might be better of as either "three separate cases" or "entirely decided within one case"... but YMMV

Reply:
We've played around a bunch with that, and it seems to only lead to problems..